admin/spec/service/check-complete.service.spec.js

```diff
@@ -55,8 +55,8 @@ describe('check-complete.service', () => {
     xit('adds a timestamp', async (done) => {
       // TODO: Relocate this test once timestamp recording occurs in application service tier
       await service.completeCheck(completedCheck)
-      const args = spy.calls.mostRecent().args[0]
-      expect(args.hasOwnProperty('receivedByServerAt')).toBe(true)
+      // const args = spy.calls.mostRecent().args[0]
```

Comment:
is that test needed?
its already excluded and most of it commented out.

Reply:
It was commented out as the linter didn't like `spy` which is not defined.